doc/api/util.md

```diff
@@ -286,13 +286,31 @@ invoke and use the result of when inspecting the object:
 ```js
 const util = require('util');
 
-const obj = { name: 'nate' };
-obj[util.inspect.custom] = function(depth) {
-  return `{${this.name}}`;
-};
```

Comment:
If we keep it, then I'd argue that we'd want to also add text saying it's a minimal example.